### DISCUSSION TITLE
TEA-DT Infrastructure Assurance Cases

### DISCUSSION THREAD

**Discussion by chrisdburr** [alan-turing-institute/AssurancePlatform#504] [General]
Please add your assurance cases to this conversation by Tuesday 25th June (End of Day). You can upload the json file by clicking on the 'Paste, drop, or click to add files' button below the text box.

Please name your file with the name of your organisation, so that other participants know which one to vote for (e.g. `alan-turing-institute.json`).

You can then vote for another project team's assurance case by using this form: https://forms.office.com/Pages/ResponsePage.aspx?id=p_SVQ1XklU-Knx-672OE-dxav9xbraJGpD_BP5Yb3HhUNVBCUlNJNEFBWDZYT0Q2MkE4OEpPVjJaMC4u

All votes need to be cast by Friday 28th June. Please note that any votes submitted after this date or for your own project will not be counted.

Winners will be notified on Monday 1st July.

**Comment by babunaz2023** [comment]
Hi Chris,

Please find my assurance case attached 
[University-of-york-Nazrul.json](https://github.com/user-attachments/files/16013983/University-of-york-Nazrul.json)
here

**Comment by clubkeano** [comment]
Hi Chris,

I have attached our case: 
[AssuranceCase_ProjectBluebird.json](https://github.com/user-attachments/files/16017962/AssuranceCase_ProjectBluebird.json)

Cheers,
Adam

**Reply by clubkeano** [reply]
@chrisdburr oops sorry wrong version, if not too late - here it is 
[alan_turing_institute.json](https://github.com/user-attachments/files/16018127/alan_turing_institute.json)

Also I don't know if it affects the UI on your end, but it looks like the "short_description" fields are correct but the "long_description" ones aren't the latest text shown on my UI

Cheers

**Comment by abilalaytekin** [comment]
[TEADT-2024-5-19T16-54-52_Surrey.json](https://github.com/user-attachments/files/16021987/TEADT-2024-5-19T16-54-52_Surrey.json)

Hello from Surrey, here is our case.
Best regards.
